code/_globalvars/lists/ambience.dm

```diff
@@ -0,0 +1,66 @@
+
+GLOBAL_LIST_INIT(medical_ambience,list('sound/ambience/ambinice.ogg'))
+
+GLOBAL_LIST_INIT(spooky_ambience,list('sound/ambience/ambimo1.ogg','sound/ambience/ambimo2.ogg','sound/ambience/ambiruin7.ogg','sound/ambience/ambiruin6.ogg',\
+										'sound/ambience/ambiodd.ogg', 'sound/ambience/ambimystery.ogg'))
+
+GLOBAL_LIST_INIT(space_ambience,list('sound/ambience/ambispace.ogg', 'sound/ambience/ambispace2.ogg', 'sound/ambience/title2.ogg', 'sound/ambience/qwerty/starlight.ogg'))
+
+GLOBAL_LIST_INIT(maint_ambience,list('sound/ambience/ambimaint1.ogg', 'sound/ambience/ambimaint2.ogg', 'sound/ambience/ambimaint3.ogg', 'sound/ambience/ambimaint4.ogg',\
+											'sound/ambience/ambimaint5.ogg', 'sound/voice/lowHiss2.ogg', 'sound/voice/lowHiss3.ogg', 'sound/voice/lowHiss4.ogg', 'sound/ambience/ambitech2.ogg'))
+
+GLOBAL_LIST_INIT(away_ambience,list('sound/ambience/ambitech.ogg', 'sound/ambience/ambitech2.ogg', 'sound/ambience/ambiruin.ogg',\
+									'sound/ambience/ambiruin2.ogg',  'sound/ambience/ambiruin3.ogg',  'sound/ambience/ambiruin4.ogg',\
+									'sound/ambience/ambiruin5.ogg',  'sound/ambience/ambiruin6.ogg',  'sound/ambience/ambiruin7.ogg',\
+									'sound/ambience/ambidanger.ogg', 'sound/ambience/ambidanger2.ogg', 'sound/ambience/ambimaint.ogg',\
+									'sound/ambience/ambiatmos.ogg', 'sound/ambience/ambiatmos2.ogg', 'sound/ambience/ambiodd.ogg'))
+
+GLOBAL_LIST_INIT(reebe_ambience,list('sound/ambience/ambireebe1.ogg', 'sound/ambience/ambireebe2.ogg', 'sound/ambience/ambireebe3.ogg'))
+
+GLOBAL_LIST_INIT(creepy_ambience,list('sound/effects/ghost.ogg', 'sound/effects/ghost2.ogg', 'sound/effects/heart_beat.ogg', 'sound/effects/screech.ogg',\
+	'sound/hallucinations/behind_you1.ogg', 'sound/hallucinations/behind_you2.ogg', 'sound/hallucinations/far_noise.ogg', 'sound/hallucinations/growl1.ogg', 'sound/hallucinations/growl2.ogg',\
+	'sound/hallucinations/growl3.ogg', 'sound/hallucinations/im_here1.ogg', 'sound/hallucinations/im_here2.ogg', 'sound/hallucinations/i_see_you1.ogg', 'sound/hallucinations/i_see_you2.ogg',\
+	'sound/hallucinations/look_up1.ogg', 'sound/hallucinations/look_up2.ogg', 'sound/hallucinations/over_here1.ogg', 'sound/hallucinations/over_here2.ogg', 'sound/hallucinations/over_here3.ogg',\
+	'sound/hallucinations/turn_around1.ogg', 'sound/hallucinations/turn_around2.ogg', 'sound/hallucinations/veryfar_noise.ogg', 'sound/hallucinations/wail.ogg'))
+
+GLOBAL_LIST_INIT(ambience_assoc,list(
+	AMBIENCE_GENERIC = GLOB.generic_ambience,
+	AMBIENCE_HOLY = GLOB.holy_ambience,
+	AMBIENCE_DANGER = GLOB.danger_ambience,
+	AMBIENCE_RUINS = GLOB.ruins_ambience,
+	AMBIENCE_ENGI = GLOB.engi_ambience,
+	AMBIENCE_MINING = GLOB.mining_ambience,
+	AMBIENCE_MEDICAL = GLOB.medical_ambience,
+	AMBIENCE_SPOOKY = GLOB.spooky_ambience,
+	AMBIENCE_SPACE = GLOB.space_ambience,
+	AMBIENCE_MAINT = GLOB.maint_ambience,
+	AMBIENCE_AWAY = GLOB.away_ambience,
+	AMBIENCE_REEBE = GLOB.reebe_ambience,
+	AMBIENCE_CREEPY = GLOB.creepy_ambience))
```

Comment:
If we are making this list ambient_sounds on areas should just become a string define and be a lookup in this list rather than storing the list on all areas.

Reply:
it’s an assoc list

Reply:
I know, Im talking about the list on every single area

Reply:
defining a list looks and sounds kind of retarded

Reply:
@PowerfulBacon it's not storing the list on all areas, it's a global variable. There's only ever once instance of it

Reply:
I know, thats why I am saying areas should reference it rather than store their own list

Reply:
areas are still just referencing the list, not storing it?

Reply:
where are they storing it